packages/themes/scss/template.scss

```diff
@@ -0,0 +1,115 @@
+
+// default color palatte when nothing is overwritten
+$primaryColor: #00237d !default;
+$textColor: #000000 !default;
+$linkColor: #000000 !default;
+$backgroundColor: #f5f5f5 !default;
+$errorMessageColor: #f44336 !default;
+
+$navbarBackgroundColor: $primaryColor !default;
+$navbarTextColor: #FFFFFF !default;
+$navbarHeight: 56px !default;
+$navbarButtonPadding: 0 10px !default;
+
+$mainFont: 'Arial', san-serif !default;
+$textFont: $mainFont !default;
+
+
+
+// theming doc starts
+a:root, a:root:hover, div:root, p:root, h1:root, h2:root, h3:root {
+  color: $textColor;
+  font-family: $mainFont;
+}
+
+nav {
+  background: $navbarBackgroundColor;
+  height: $navbarHeight;
+  align-content: center;
+
+  a, div, p, h1, h2, h3 {
+    color: $navbarTextColor;
+    padding: $navbarButtonPadding;
+  }
+}
+
+html,
+body{
+  background: $backgroundColor;
+}
+
+main {
+  background-color: $backgroundColor;
+  margin-top: $navbarHeight;
+}
+
+
+// Material UI
+.mat-error {
+  color: $errorMessageColor;
+  font-size: 12px;
+}
+
+.mat-form-field-invalid .mat-input-element, .mat-warn .mat-input-element {
+  caret-color: $errorMessageColor;
+}
+
+.mat-input-element {
+  caret-color: $textColor;
+}
+
+.dv-mat-button {
+  display: inline;
+}
+
+// About mat-button
+// there are three ways to overwrite the button information
+//  1. Have a wrapper class called "dv-mat-button" on every
```

Comment:
Can you get the same result if you add the `dv-mat-button` class to the button instead of to the div that wraps it?

Reply:
I tried it but unfortunately it didn't work. :(
I think the reason is that I can't lower the scope by calling an element with two classes.

Reply:
OK, thanks for checking